### DISCUSSION TITLE
On/Off state for lights that do not have brightness attribute?

### DISCUSSION THREAD

**Discussion by ghost** [nicufarmache/lovelace-big-slider-card#45] [Q&A]
Is there a way to display "state" instead of brightness %? 

Current yaml for that device:

type: custom:big-slider-card
entity: light.game_room
show_percentage: true
name: Main Light
attribute: brightness
icon: mdi:chandelier
bold_text: true
min: 0
max: 100
colorize: false
min_slide_time: 2
icon_color: '#EEB76B'
height: 90
background_color: var(--ha-card-background)

![Screenshot 2023-12-13 151719](https://github.com/nicufarmache/lovelace-big-slider-card/assets/23648025/07a5dc63-2ac0-4349-91db-20afa5d2d577)

**Comment by nicufarmache** [comment]
Hey, I'm currently working on this feature. It will probably be available in the next version
